kore/src/main/scala/org/kframework/attributes/Att.scala

```diff
@@ -108,9 +118,21 @@ object Att {
   val UNIQUE_ID = "UNIQUE_ID"
 
   private val stringClassName = classOf[String].getName
+  private val intClassName = classOf[java.lang.Integer].getName
 
   def from(thatAtt: java.util.Map[String, String]): Att =
     Att(immutable(thatAtt).map { case (k, v) => ((k, Att.stringClassName), v) }.toMap)
+
+  private def apply(thatAtt: Map[(String, String), Any]) = {
+    new Att(thatAtt)
+  }
+
+  def mergeAttributes(p: Set[Att]) = {
```

Comment:
Isn't there a standard operator that can be overridden? Maybe `:+`?

Reply:
Maybe, but I see no reason to make it harder to call from Java code.